docs/content/docs/ops/state/savepoints.md

```diff
@@ -185,10 +185,86 @@ This submits a job and specifies a savepoint to resume from. You may give a path
 
 By default the resume operation will try to map all state of the savepoint back to the program you are restoring with. If you dropped an operator, you can allow to skip state that cannot be mapped to the new program via `--allowNonRestoredState` (short: `-n`) option:
 
+#### Restore mode
```

Comment:
Another thing to consider is whether we should move this to a separate document or keep it here.